src/main/java/io/qameta/allure/AllureConstants.java

```diff
@@ -18,6 +21,12 @@
 
     public static final String TEST_RUN_JSON_FILE_GLOB = "*-testrun.json";
 
+    public static final String ATTACHMENT_FILE_SUFFIX = "-attachment";
+
+    public static final Charset ATTACHMENT_ENCODING = StandardCharsets.UTF_8;
+
+    public static final Integer MAX_TITLE_LENGTH = 120;
```

Comment:
As I sad before we don't need to limit titles anymore, so we need to remove this and all related functionality